README.md

```diff
@@ -67,9 +67,10 @@ Go to [Get Started](./docs/tutorial/get-started.md) page to try out Vald :)
     - gRPC APIs can be triggered by any programming languages which support gRPC.
     - REST API is also supported.```
 
-## Requirement
+## Requirements
 
-kubernetes 1.17~
+- Kubernetes 1.17~
+- AVX2 instructions (required by Vald Agent NGT)
```

Comment:
I have the following question.

1. Does `X86` supported?
2. ARM CPU supported?
3. Only AVX2 is required? other AVX instruction is required?

```suggestion
- X64 CPU with AVX2 instructions set supported (required by Vald Agent NGT)
```

Reply:
1 and 2 ... currently, we don't provide docker images for these CPU architectures.
however, we are planning to provide docker images for each of them in the near future.

3 ... i'm not sure but maybe only AVX2 is needed. @kpango do you know about that?

Reply:
BTW not sure if AVX is required or optional 🤔

Reply:
Currently, AVX2 is required, but once the issue #687 is resolved, it becomes optional.